packages/venia-ui/lib/components/CartPage/GiftCards/giftCards.js

```diff
@@ -122,7 +122,11 @@ const GiftCards = props => {
                 id={classes.card}
                 label="Gift Card Number"
             >
-                <div className={classes.card_input_container}>
+                <div
+                    className={`${classes.card_input_container} ${
+                        shouldDisplayCardError ? classes.error : ''
```

Comment:
@tjwiebell No, I definitely overlooked it. You're right, we avoid classname concatenation in all cases.

As for your suggestion, I don't recommend using the `:not` pseudo selector.

Reply:
All cases should now use modifiers, with the exception of `filterDefault.js`. Those checkboxes should really be `<input type="checkbox">`, instead of `<button>`. Semantics of filters will be addressed in upcoming story PWA-713.